packages/suite-web/src/support/Router.tsx

```diff
@@ -144,23 +144,20 @@ const components: Record<PageName, LazyExoticComponent<ComponentType<any>>> = {
 };
 
 const AppRouter = () => (
-    // inititating strict mode higher would throw an error from react-helmet
```

Comment:
`StrictMode` can now be moved higher up.

Reply:
done